<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QAction{color:white;}
QMainWindow{color:white;background-color:black;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>1311</width>
      <height>641</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget{background-color:white}
QPushButton{background-color:lightgrey}</string>
    </property>
    <widget class="QPushButton" name="submitButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>310</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="newNameInput">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>200</y>
       <width>271</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit{background-color:lightgrey}</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>200</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Enter the new name of location:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>40</y>
       <width>531</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>36</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stage Editor</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>390</y>
       <width>351</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Or select a previous location:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="openDropDown">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>400</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QComboBox{background-color:lightgrey}</string>
     </property>
    </widget>
    <widget class="QPushButton" name="openButton">
     <property name="geometry">
      <rect>
       <x>1030</x>
       <y>500</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteButton">
     <property name="geometry">
      <rect>
       <x>860</x>
       <y>500</y>
       <width>121</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="creatingSquareLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white}</string>
    </property>
    <property name="text">
     <string>Press the location of one of the corners of the square</string>
    </property>
   </widget>
   <widget class="QLabel" name="creatingSquareLabel_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{color:white}</string>
    </property>
    <property name="text">
     <string>Now Press the location of the final coordinate of the square</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1900</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <widget class="QMenu" name="menuAdd_new_bar">
     <property name="title">
      <string>Add new bar</string>
     </property>
     <addaction name="actionHorizontal"/>
     <addaction name="actionVertical"/>
    </widget>
    <addaction name="menuAdd_new_bar"/>
    <addaction name="actionAddStageSquare"/>
   </widget>
   <widget class="QMenu" name="menuopen">
    <property name="title">
     <string>Open</string>
    </property>
    <addaction name="actionOpen_location"/>
   </widget>
   <addaction name="menuAdd"/>
   <addaction name="menuopen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionHorizontal">
   <property name="text">
    <string>horizontal</string>
   </property>
  </action>
  <action name="actionVertical">
   <property name="text">
    <string>vertical</string>
   </property>
  </action>
  <action name="actionOpen_location">
   <property name="text">
    <string>Open location</string>
   </property>
  </action>
  <action name="actionAddStageSquare">
   <property name="text">
    <string>Add Stage Square</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
